client/src/component/Uploader/FileUploaderDialog.js

```diff
@@ -0,0 +1,59 @@
+
+import Dialog from "@material-ui/core/Dialog";
+import React, { useState } from "react";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dropzone from "react-dropzone";
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Button from "@material-ui/core/Button";
+
+
+const FileUploaderDialog = (props) => {
+  return (
+    <React.Fragment>
```

Comment:
Personal preference here, but the new syntax for `Fragment` is a lot less verbose: `<></>` 😛